tests/021.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+preg_replace() should untaint a string
+--SKIPIF--
+<?php if (!extension_loaded("taint")) print "skip"; ?>
+--INI--
+taint.enable=1
+--FILE--
+<?php
+class request {
+    function get() {
+        $a = "a" ."";
```

Comment:
@laruence could you give me a hint whether this is expected behaviour or what your opinion is on this topic?